includes/Integrations/Jetpack.php

```diff
@@ -274,7 +279,7 @@ public function filter_rest_api_response( WP_REST_Response $response, WP_Post $p
 
 		$metadata = wp_get_attachment_metadata( $post->ID );
 
-		if ( $metadata && isset( $metadata['videopress']['duration'] ) ) {
+		if ( $metadata && isset( $metadata['videopress']['duration'], $data['media_details'] ) && is_array( $data['media_details'] ) ) {
```

Comment:
```suggestion
		if ( $metadata && isset( $metadata['videopress']['duration'] ) ) {
		   if( ! isset( $data['media_details']  ) || ! is_array( $data['media_details']  )  ) {
		        $data['media_details'] = [];
		   }
```

Reply:
Just a readability preference?

Reply:
This would mean a change in behavior ⚠️ 

Right now, the code only adds duration to `media_details` if `media_details` is actually a requested field. Otherwise `media_details` is not provided.

Your suggested change would make it so that `media_details` is always provided.